src/Rollbar.php

```diff
@@ -247,7 +286,7 @@ public static function destroy()
     public static function report_exception($exc, $extra_data = null, $payload_data = null)
```

Comment:
Does it make sense to annotate the return type here as we're doing with the rest of methods?

Reply:
I am planning on adding type annotations and DocBlocks to most everything. I will submit that in a separate PR.